homeassistant/components/vizio/config_flow.py

```diff
@@ -51,6 +67,13 @@ def _get_config_schema(input_dict: Dict[str, Any] = None) -> vol.Schema:
             vol.Optional(
                 CONF_ACCESS_TOKEN, default=input_dict.get(CONF_ACCESS_TOKEN, "")
             ): str,
+            vol.Optional(
+                CONF_INCLUDE_OR_EXCLUDE, default=default_include_or_exclude.title(),
+            ): vol.In([CONF_INCLUDE.title(), CONF_EXCLUDE.title()]),
+            vol.Optional(
```

Comment:
Wouldn't it make more sense to move this to a 2nd page of the config flow, and only show if it's a TV?

Reply:
Yes my plan was to make this change in a second PR to reduce the change scope since `CONF_ACCESS_TOKEN` should also be on the second page since it's only relevant for TVs, but I can add that to this one

Reply:
Actually will keep `CONF_ACCESS_TOKEN` on the first page since the first page is about checking and verifying connectivity